### DISCUSSION TITLE
Mime handling

### DISCUSSION THREAD

**Discussion by NegaScout** [regolith-linux/regolith-desktop#884] [Q&A]
Hi, 
I would like to configure `ssh://` handling, but not sure where to begin. i have tried modifying `.config/mimeapps.list` to
```
[Added Associations]
application/vnd.ms-publisher=org.gnome.gedit.desktop;
application/pdf=firefox.desktop;
application/pkix-cert=org.gnome.gedit.desktop;
application/vnd.apple.keynote=org.gnome.gedit.desktop;
x-scheme-handler/ssh=ssh-terminal.desktop
[Default Applications]
application/pkix-cert=org.gnome.gedit.desktop
application/vnd.apple.keynote=org.gnome.gedit.desktop
text/html=firefox_firefox.desktop
x-scheme-handler/http=firefox_firefox.desktop
x-scheme-handler/https=firefox_firefox.desktop
x-scheme-handler/about=firefox_firefox.desktop
x-scheme-handler/unknown=firefox_firefox.desktop
```
but it does not seem to work. Where can I configure such behaviour? I should be on latest Regolith 3 on X11.
Thank you!

**Comment by kgilmer** [comment]
Hi @NegaScout , I'm not familiar w this..can you briefly describe what you want to happen in your system?

**Reply by NegaScout** [reply]
Sure! I want to click on `ssh://<ip_address>` link in browser and the system opening new terminal with and ssh me to that address. Its the same, as opening link in browser, as `ssh` in this is a protocol, the same as `http`/`https` and coresponding fqdn/ip
